### DISCUSSION TITLE
How much of a URL is needed

### DISCUSSION THREAD

**Discussion by Coderdude112** [keepassxreboot/keepassxc#6147] [Q&A] [answered]
Hello!

I am switching to Keepassxc from Lastpass and have imported all of my things. When I am specifying a URL exactly how much is necessary? I have some URLs that are really really long (Specifying the exact login page) but looking at the user guide the Netflix example for Netflix specifies the URL as `https://www.netflix.com/` is this all that is needed?

Thanks for your help in advance!!

**Comment by droidmonkey** [comment, marked as answer]
All you need is the base, even https://netflix.com is sufficient. Adding a subdomain makes it "more specific" and will not match other subdomains. For example: https://sub1.netflix.com and https://sub2.netflix.com would be entirely distinct when returning credentials. Everything after the base does not matter unless you enable "Only return best matching credentials" in the browser setttings.

**Reply by Coderdude112** [reply]
Thanks for your quick reply!

Have an amazing day!!!
